types/node/index.d.ts

```diff
@@ -5036,6 +5038,7 @@ declare module "stream" {
             wrap(oldStream: NodeJS.ReadableStream): Readable;
             push(chunk: any, encoding?: string): boolean;
             destroy(error?: Error): void;
+            _destroy(error: Error | null, cb: (e: Error | null) => any): void;
```

Comment:
actually `_destroy` gets the error passed to `destroy` so I think here we have and exception to the `Error | null` rule.
And I think the return type of `cb` should be `void`.

Reply:
destroy contains the call `this._destroy(error || null)` - https://github.com/mcollina/node/blob/master/lib/internal/streams/destroy.js#L32 .

Reply:
Ah yes, you are right.